src/constants.F90

```diff
@@ -219,6 +219,9 @@ module constants
        N_D0    = 650, N_DC    = 699, N_T0   = 700, N_TC    = 749, N_3HE0  = 750, &
        N_3HEC  = 799, N_A0    = 800, N_AC   = 849, N_2N0   = 875, N_2NC   = 891
 
+  ! Depletion reactions
+  integer, parameter :: DEPLETION_RX(6) = [N_2N, N_3N, N_4N, N_GAMMA, N_P, N_A]
```

Comment:
Why these?  In the massive tally problem you have are these the ones responsible for most of the x/s lookup time (on develop)?  Are there others that, in that problem, might be worthwhile to add?  Or, are there some here (probably N_4N) that do not add enough savings to justify the cost?

Reply:
These are the reactions (excluding fission) that account for almost all of the transmutation. Fission is not included because its cross section is already calculated by default. It does become questionable as to how much some of these reactions contribute, but these are the reactions that have typically been included in a number of traditional depletion solvers (ORIGEN2 and MCODE are the ones I'm familiar with and this is the set of reactions they model). I suppose in the future we may want to consider having this as a configurable option.